wallet-web/components/bond/BondNodeForm.tsx

```diff
@@ -120,59 +134,49 @@ export default function BondNodeForm(props: TBondNodeFormProps) {
     identityKey,
     host,
     version,
-    verlocPort,
     location,
-    mixPort,
-    httpApiPort,
-    clientsPort,
-  }: TFormInput): boolean => {
-    let validAmount = validateAmount(amount)
-    let validSphinxKey = validateKey(sphinxKey)
-    let validIdentityKey = validateKey(identityKey)
-    let validHost = validateHost(host)
-    let validVersion = validateVersion(version)
-
-    let validLocation =
-      props.type == NodeType.Gateway ? validateLocation(location) : true
-
+    verlocPort = DEFAULT_VERLOC_PORT.toString(),
+    mixPort = DEFAULT_MIX_PORT.toString(),
+    httpApiPort = DEFAULT_HTTP_API_PORT.toString(),
+    clientsPort = DEFAULT_CLIENTS_PORT.toString(),
```

Comment:
If these values are undefined i.e. the advanced area is not displayed, then we fallback to our default values.